ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -148,7 +150,7 @@ objects:
 
         i=0
         MAX_RETRIES=10
-        until oc wait deploy/router -n "${ROUTER_NAMESPACE}" --for condition=available --timeout=10m || [ $i -eq $MAX_RETRIES ]; do
+        until oc wait ${ROUTER_DEPLOYMENT} -n "${ROUTER_NAMESPACE}" --for condition=available --timeout=10m || [ $i -eq $MAX_RETRIES ]; do
```

Comment:
quotes are not required?
`"${ROUTER_NAMESPACE}"` has quotes...

Reply:
well quotes would be required if there's spaces in the actual name of the object (its `router[-default]` currently) I can put those in.